source/content/guides/launch/05-redirects.md

```diff
@@ -12,26 +12,28 @@ pagination:
 use:
     - launchpages
 permalink: docs/guides/launch/redirects/
-nexturl: guides/launch/launch-check/
```

Comment:
Not sure, why these are being deleted

Reply:
Because they are no longer needed in Gatsby.